### PR TITLE
Add specs to validate Github CTB

### PR DESCRIPTION
**Pull request by praveen4g0**
* Added more tests to validate github CTB

### CODE REVIEW

**Comment by praveen4g0** [comment]
```gauge run --env "default, test" --log-level=debug --verbose   specs/triggers/eventlistener.spec```

you can run this on any ocp cluster with pipelines  1.1 installed

File: testdata/triggers/github-ctb/eventlistener-ctb-git-pr.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  triggers:
+    - name: foo-trig
+      bindings:
+        - ref: github-pullreq
+          kind: ClusterTriggerBinding
+        - name: message-binding
+          spec:
+            params:
+              - name: MESSAGE
+                value: Hello from the Triggers your triggering (github CTB) of event-type pull-request!
```
Comment (VeereshAradhya) [inline testdata/triggers/github-ctb/eventlistener-ctb-git-pr.yaml:17]: `your` -> `you are`

File: testdata/triggers/github-ctb/eventlistener-ctb-git-push.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  triggers:
+    - name: foo-trig
+      bindings:
+        - ref: github-push
+          kind: ClusterTriggerBinding
+        - name: message-binding
+          spec:
+            params:
+              - name: MESSAGE
+                value: Hello from the Triggers your triggering (github CTB) of event-type push!
```
Comment (VeereshAradhya) [inline testdata/triggers/github-ctb/eventlistener-ctb-git-push.yaml:17]: Hello from the Triggers, you are triggering

Comment (praveen4g0) [reply testdata/triggers/github-ctb/eventlistener-ctb-git-push.yaml]: Resolved

**Review by VeereshAradhya — APPROVED** [review]

**Pull request merged**